tst/EnergyPlus/unit/HVACStandaloneERV.unit.cc

```diff
@@ -0,0 +1,95 @@
+// EnergyPlus::Standalone ERV Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include <EnergyPLus/HVACStandAloneERV.hh>
+#include <EnergyPlus/DataHeatBalance.hh>
+#include <EnergyPlus/DataZoneEquipment.hh>
+#include <EnergyPlus/DataSizing.hh>
+#include <EnergyPlus/UtilityRoutines.hh>
+#include <ObjexxFCL/gio.hh>
+#include <EnergyPlus/SQLiteProcedures.hh>
+
+
+using namespace EnergyPlus;
+using namespace EnergyPlus::HVACStandAloneERV;
+using namespace ObjexxFCL;
+using namespace EnergyPlus::DataHeatBalance;
+using namespace DataGlobals;
+using namespace EnergyPlus::DataZoneEquipment;
+using namespace EnergyPlus::DataSizing;
+
+
+TEST( SizeStandAloneERVTest, Test1 )
+{
+
+	int write_stat;
+	// Open the Initialization Output File (lifted from SimulationManager.cc)
+	OutputFileInits = GetNewUnitNumber( );
+	{ IOFlags flags; flags.ACTION( "write" ); flags.STATUS( "UNKNOWN" ); gio::open( OutputFileInits, "eplusout.eio", flags ); write_stat = flags.ios( ); }
+//	eso_stream = gio::out_stream( OutputFileStandard );
+
+	//CreateSQLiteDatabase(); (lifted from SimulationManager.cc)
+	try {
+		EnergyPlus::sqlite = std::unique_ptr<SQLite>( new SQLite( ) );
```

Comment:
@mjwitte At the moment, no there is no way to set writeOutputToSQLite() to false without doing what you did here.

Basically what happens when you construct a 'new SQLite()' it currently looks for SQLite in the input file. If it doesn't find it then writeOutputToSQlite() is false and no database is created or used.

I am currently refactoring SQLite to unit test it more easily and I will keep this issue in mind as I refactor SQLite.